### ISSUE TITLE
Movinet finetune on Kintics600

### ISSUE THREAD

**Issue by Wangbk-dl** [tensorflow/models#11089]
I'm finetuning the movinet on Kinetics600 dataset. When I load the a0-stream checkpoint, choose 35 target classes that I'm interested and evaluate the precise of the loaded model, but I found that there was only 45.41% top-1 acc. I have not done any fintune yet. So I guess the result comes from the poor data pre-process. I didn't do anything special, but only normalize the pixels from 0-255 to 0-1 without any data-augmentation.
I didn't find any instructions about how to augment the training data, so I came here for some help.
Anyone could help me with that?

**Comment by laxmareddyp** [comment]
Hi @Wangbk-dl ,

Thanks for reaching out here, but unfortunately, we do not have any document instructions on how to do augment the training data. You have the option to train and fine-tune your own custom dataset using the [movinet_streaming_model_training_and_inference](https://github.com/tensorflow/models/blob/master/official/projects/movinet/movinet_streaming_model_training_and_inference.ipynb) provided. This allows you to adapt the movinet model to your specific dataset and tasks by training it from scratch or fine-tuning on your data.This tutorial might helpful to you.

Thanks.
